test/components/tree-view.spec.tsx

```diff
@@ -215,6 +215,22 @@ describe('<TreeView />', () => {
         });
 
         describe('Keyboard Navigation', () => {
+            it('reacts to keyboard events even when a node is not focused initially', async () => {
+                const {driver: treeViewDemo, waitForDom} = clientRenderer.render(
+                    <TreeViewDemo />
```

Comment:
I tried it for a bit and it seems this test could be a regular unit test and not a demo test.